config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/EncassEntityBuilder.java

```diff
@@ -91,6 +91,7 @@ private Entity buildEncassEntity(AnnotatedBundle annotatedBundle, Bundle bundle,
         String guid = encass.getGuid();
         String id = encass.getId();
         AnnotatedEntity annotatedEntity = annotatedBundle != null ? annotatedBundle.getAnnotatedEntity() : null;
+        final boolean isRedeployableBundle = annotatedEntity != null && annotatedEntity.isBundle() && annotatedEntity.isRedeployable();
```

Comment:
isBUndle check is not required entity present in annotatedBundle is always a bundle annotation entity

Reply:
done